Modules/_testlimitedcapi/set.c

```diff
@@ -174,6 +219,8 @@ static PyMethodDef test_methods[] = {
     {"set_clear", set_clear, METH_O},
 
     {"test_frozenset_add_in_capi", test_frozenset_add_in_capi, METH_NOARGS},
+    {"test_set_contains_does_not_convert_unhashable_key",
+     test_set_contains_does_not_convert_unhashable_key, METH_NOARGS},
```

Comment:
No I don't need to call it; because it's added to `test_methods` there.

You can check that it's called by modifying it and running:

```
python -E  -m test test.test_capi
```

This is automatically discovered here:

https://github.com/python/cpython/blob/f963864cb54c2e7364b2c850485c6bf25479f6f2/Lib/test/test_capi/test_misc.py#L2648-L2649